### PR TITLE
fix less command on amazon linux 2

### PR DESCRIPTION
**Pull request by garcialuis**
#### What type of PR is this?

type::bug

#### What this PR does / why we need it:
This PR is to fix the less command on amazon linux 2 after installing kURL. This will also fix existing installations with the bug once the installer is rerun.

#### Which issue(s) this PR fixes:

Fixes # [SC-43509](https://app.shortcut.com/replicated/story/43509/kurl-installation-in-amazon-linux-2-breaks-less-command)

## Steps to reproduce

Details to reproduce the issue are listed on the SC ticket linked above. 

#### Does this PR introduce a user-facing change?

```release-note
NONE
```
#### Release note:
```release-note
- Fixes a bug where the less command breaks after installing kURL on amazon linux 2.
```

#### Does this PR require documentation?

NONE

### CODE REVIEW

File: scripts/common/kubernetes.sh [hunk 1]
```diff
@@ -92,6 +92,11 @@ function kubernetes_install_host_packages() {
 
     if kubernetes_host_commands_ok "$k8sVersion"; then
         logSuccess "Kubernetes host packages already installed"
+        # less command is broken if libtinfo.so.5 is missing in amazon linux 2
+        if [ "$LSB_DIST" == "amzn" ] && ! file_exists "/usr/lib64/libtinfo.so.5"; then
+            install_host_packages "${DIR}/packages/kubernetes/${k8sVersion}" ncurses-compat-libs
```
Comment (emosbaugh) [inline scripts/common/kubernetes.sh:97]: Are you sure you still need this?

File: testgrid/specs/full.yaml [hunk 2]
```diff
@@ -1450,3 +1450,12 @@
       shouldDisableRebootServices: false
       shouldDisableClearNodes: false
       shouldEnablePurgeNodes: false
+- name: less_command
+  installerSpec:
+    kubernetes:
+      version: "1.23.5"
+    docker:
+      version: "20.10.5"
+  postInstallScript: |
```
Comment (emosbaugh) [inline testgrid/specs/full.yaml:1459]: Did you run this to see if it fails prior to the fix?

File: scripts/common/kubernetes.sh (hunk 1 above)
Comment (emosbaugh) [reply scripts/common/kubernetes.sh:97]: ah i see for existing installs

**Review by emosbaugh — APPROVED** [review]

File: scripts/common/kubernetes.sh (hunk 1 above)
Comment (garcialuis) [reply scripts/common/kubernetes.sh:97]: Yeah, I left that in for existing installs.

File: testgrid/specs/full.yaml (hunk 2 above)
Comment (emosbaugh) [reply testgrid/specs/full.yaml:1459]: Would it be red and say fail in the browser? If so yes

**Pull request merged**